onmt/decoders/transformer.py

```diff
@@ -88,7 +91,10 @@ def forward(self, inputs, memory_bank, src_pad_mask, tgt_pad_mask,
 
         query_norm = self.layer_norm_2(query)
         mid, attn = self.context_attn(memory_bank, memory_bank, query_norm,
-                                      mask=src_pad_mask)
+                                      mask=src_pad_mask,
+                                      layer_cache=None,
```

Comment:
Why `None`? Looks like you need to update your numbers. :)

Reply:
Thanks... Just updated them. Hope this fits more your expectations :)